### DISCUSSION TITLE
fps overlay is always visible

### DISCUSSION THREAD

**Discussion by dancer69** [Heroic-Games-Launcher/HeroicGamesLauncher#855] [Q&A] [answered]
I have a problem with heroic games with the fps overlay, which is always enabled and visible in game's left corner. Most recent game I tried was Shadow of the Tomb Raider. I'm using mangohud overlay, but this other fps overlay doesn't go away. I have set in both general and game's settings "Show fps(Dx9,10,11)" to false and also I checked the config file in ~/.config/heroic folder to see if somehow the setting doesn't apply. I'm using the latest heroic version on arch linux. Do I need to do something else to remove it?
EDIT:
Ignore my question(I cannot delete it). After I posted it I figure it out. I forgot to check legendery config folder and there was the problem. It was enabled as env variable in config.ini.

**Comment by flavioislima** [comment, marked as answer]
Check your bash profile file to see if you have set DXVK_HUD there or on your env.
Probably you have it set globally.
If you want to force it off on heroic add this environment variable to the game settings:
DXVK_HUD=0

**Reply by dancer69** [reply]
Yes, that was the problem. I've already disable it in legendary's config file in which was enabled.

**Comment by flavioislima** [comment]
Nice
